earthpy/tests/test_spatial.py

```diff
@@ -108,6 +109,63 @@ def test_extent_to_json():
         es.extent_to_json([0, 1, 1, 0])
 
 
+def test_normalized_diff_shapes():
+    """If provided with arrays of different shapes,
+    normalized_diff returns ValueError."""
+
+    # Test data
+    b1_2d = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
+    b2_1d = np.array([[1, 2, 3, 4, 5, 14, 12, 13, 14, 17]])
+
+    # Check ValueError
+    with pytest.raises(ValueError, match=r"Both arrays should be of the same dimensions"):
```

Comment:
What about saying `"Both arrays should have the same dimensions"`  both here and in the function err too?

Reply:
The text for this ValueError has now been modified in the function and the test.

Reply:
This may be a silly question, but is the raw string literal necessary here for the value error message matching?

Reply:
@mbjoseph @joemcglinchy Not sure if there is a better way, but when I use `message = ` rather than `match=r`, I can change the text to not match, and the test still passes.

Reply:
Does the test work if you still use the `match` argument without the raw string literal? e.g., 

```python
with pytest.raises(ValueError, match="Both arrays should be of the same dimensions"):
```

Reply:
Good call. The test works without the `r`, so I have removed it now. Thanks, Max!